pkg/prometheus/prometheus.go

```diff
@@ -80,6 +82,52 @@ func getRouter(registry Registry, path string) (*http.ServeMux, error) {
 	return mux, nil
 }
 
+// GetRegistry returns a fully configured Prometheus registry with metrics based on Kube Linter validations
+func GetRegistry() (*prometheus.Registry, error) {
```

Comment:
nit: Is `GetRegistry` a good name here? Perhaps it looks more like initialization or setup of registry...

Reply:
Yes, this will change on next refactor. I "copied" before method but will definitely change it to a more suitable name.